src/devices/Common/System/Device/Analog/ValueChangedEventArgs.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using UnitsNet;
+
+namespace System.Device.Analog
+{
+    /// <summary>
+    /// Arguments passed in when an event is triggered by the GPIO.
+    /// </summary>
+    public class ValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ValueChangedEventArgs"/> class.
+        /// </summary>
+        /// <param name="rawValue">The raw analog sensor reading</param>
+        /// <param name="newValue">The analog sensor reading, converted to voltage.</param>
+        /// <param name="pinNumber">The pin number that triggered the event.</param>
+        /// <param name="triggerReason">The reason for the event</param>
+        public ValueChangedEventArgs(uint rawValue, ElectricPotential newValue, int pinNumber, TriggerReason triggerReason)
+        {
+            RawValue = rawValue;
+            Value = newValue;
+            PinNumber = pinNumber;
+            TriggerReason = triggerReason;
+        }
+
+        /// <summary>
+        /// Raw value, unscaled.
+        /// </summary>
+        public uint RawValue
```

Comment:
I'm a bit sceptical of this being 32-bit value. For most ADCs this will work but I'm not sure this will really scale that well. Also note that RawValue is meaningless unless you also have MaxValue

Reply:
The MaxValue is a property of the pin, so I'm not sure we should provide it basically as a duplicate. Some people might want the raw value (for whatever reason), so I think removing it wouldn't be helpful.